Makefile

```diff
@@ -90,3 +90,20 @@ changelog:
 
 mypy:
 	poetry run mypy --pretty aws_lambda_powertools
+
+format-examples:
+	poetry run isort docs/examples
+	poetry run black docs/examples/*/*.py
+	poetry run black docs/examples/*/*/*.py
+	poetry run black docs/examples/*/*/*/*.py
+	poetry run black docs/examples/*/*/*/*/*.py
+
+lint-examples:
+	poetry run python3 -m py_compile docs/examples/*/*.py
+	poetry run python3 -m py_compile docs/examples/*/*/*.py
+	poetry run python3 -m py_compile docs/examples/*/*/*/*.py
+	poetry run python3 -m py_compile docs/examples/*/*/*/*/*.py
+	cfn-lint docs/examples/*/*.yml
```

Comment:
`cfn-lint` would have to be added as a dependency if we want to keep this check in place